dist/elasticui.js

```diff
@@ -912,12 +925,12 @@ var elasticui;
                     var directive = {};
                     directive.restrict = 'E';
                     directive.scope = true;
-                    directive.template = '\
-            <ul class="pager">\
-                <li ng-if="indexVM.page <= 1" class="disabled"><a href="">Previous</a></li>\
-                <li ng-if="indexVM.page > 1"><a href="" ng-click="indexVM.page=indexVM.page - 1">Previous</a></li>\
-                <li ng-if="indexVM.pageCount <= indexVM.page" class="disabled"><a href="">Next</a></li>\
-                <li ng-if="indexVM.pageCount > indexVM.page"><a href="" ng-click="indexVM.page=indexVM.page + 1">Next</a></li>\
+                    directive.template = '\
+            <ul class="pager">\
+                <li ng-if="indexVM.page <= 1" class="disabled"><a href="">Previous</a></li>\
+                <li ng-if="indexVM.page > 1"><a href="" ng-click="indexVM.page=indexVM.page - 1">Previous</a></li>\
+                <li ng-if="indexVM.pageCount <= indexVM.page" class="disabled"><a href="">Next</a></li>\
+                <li ng-if="indexVM.pageCount > indexVM.page"><a href="" ng-click="indexVM.page=indexVM.page + 1">Next</a></li>\

```

Comment:
The only changes in this template are whitespace changes? Why was this necessary?